backfill/2022-04-27-firefox-desktop-glean-rejected/02-gcp-ingestion-beam-decoder.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -exo pipefail
+
+# This script assumes it's being run from the ingestion-beam directory of the gcp-ingestion repo
+# with the code modified to convert any events[*].timestamp float values into integers.
+
+PROJECT="moz-fx-data-backfill-10"
+JOB_NAME="firefox-desktop-backfill"
+
+mvn \
+  clean \
+  compile \
+  exec:java -Dexec.mainClass=com.mozilla.telemetry.Decoder -Dexec.args="\
+      --runner=Dataflow \
+      --jobName=$JOB_NAME \
+      --project=$PROJECT \
+      --geoCityDatabase=gs://moz-fx-data-prod-geoip/GeoIP2-City/20220426/GeoIP2-City.mmdb \
+      --geoIspDatabase=gs://moz-fx-data-prod-geoip/GeoIP2-ISP/20220426/GeoIP2-ISP.mmdb \
+      --geoCityFilter=gs://moz-fx-data-prod-dataflow-templates/cities15000.txt \
+      --schemasLocation=gs://moz-fx-data-prod-dataflow/schemas/202204250316_17084144.tar.gz \
+      --inputType=bigquery_table \
+      --input=\"$PROJECT:payload_bytes_error.structured\" \
+      --bqReadMethod=storageapi \
+      --outputType=bigquery \
+      --bqWriteMethod=file_loads \
+      --bqClusteringFields=submission_timestamp \
+      --output=$PROJECT:\${document_namespace}_live.\${document_type}_v\${document_version} \
+      --errorOutputType=bigquery \
+      --errorOutput=$PROJECT:payload_bytes_error.backfill \
```

Comment:
I seem to remember @whd maybe wanting to change the setup slightly, where `backfill` would have the copied error records to be reprocessed and any errors during the reprocessing would go into `structured`?

Reply:
I did want to do this, but forgot to. I just added it to the cloudops-infra PR, so this and future backfills should source their data from `backfill` and write their error output to their standard pbe table.